src/components/NewWorkspaceModal.ts

```diff
@@ -362,8 +370,17 @@ const NewWorkspaceModal = withDisplayName(
       return !value ? '' : `Option ${value['aria-label']} selected.`;
     };
 
+    const getNoApplicableBillingProjectsMessage = () => {
+      if (cloneWorkspace) {
+        return 'You do not have a billing project that is able to clone this workspace.';
+      }
+      return requireEnhancedBucketLogging
+        ? 'You do not have access to a billing project with additional security monitoring.'
+        : 'You need a billing project to create a new workspace.';
+    };
+
     return Utils.cond(
-      [loading || billingProjects === undefined, () => spinnerOverlay],
```

Comment:
The default value of `billingProjects` is [], so the check on `billingProjects` could never be undefined.

Note that the check for `hasBillingProjects` at line 385 determines whether the full dialog is shown, or the simple one that says you have no billing projects (example below).

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/d2dba162-30d4-45dd-a1a5-6e231819c3b2)